x-pack/plugins/observability_solution/slo/public/embeddable/slo/overview/slo_embeddable_factory.tsx

```diff
@@ -56,6 +68,14 @@ export const getOverviewEmbeddableFactory = (deps: SloEmbeddableDeps) => {
         {
           ...titlesApi,
           defaultPanelTitle: defaultTitle$,
+          getTypeDisplayName: () =>
+            i18n.translate('xpack.slo.editSloOverviewEmbeddableTitle.typeDisplayName', {
+              defaultMessage: 'criteria',
+            }),
+          isEditingEnabled: () => api.getSloGroupOverviewConfig().overviewMode === 'groups',
```

Comment:
@ThomThomson Before I was using a custom action with an [isCompatible](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/slo/public/ui_actions/edit_slo_overview_panel.tsx#L81) method. Now that I removed the custom action I check only for the overviewMode to decide if editing is enabled, which is specific to the logic of the embeddable. 

Do I need to do the rest checks I was doing [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/slo/public/ui_actions/edit_slo_overview_panel.tsx#L35) as well?

Reply:
No you don't need to do those checks anymore! Those checks were there to prevent _other_ embeddables from ending up with the `edit SLO overview` action, but now that it's using a generic action that isn't a risk.